x-pack/plugins/security_solution/public/common/components/events_viewer/index.tsx

```diff
@@ -153,11 +154,11 @@ const StatefulEventsViewerComponent: React.FC<Props> = ({
 
   const globalFilters = useMemo(() => [...filters, ...(pageFilters ?? [])], [filters, pageFilters]);
   const trailingControlColumns: ControlColumnProps[] = EMPTY_CONTROL_COLUMNS;
-  const graphOverlay = useMemo(
-    () =>
-      graphEventId != null && graphEventId.length > 0 ? <GraphOverlay timelineId={id} /> : null,
-    [graphEventId, id]
-  );
+  const graphOverlay = useMemo(() => {
+    const shouldShowOverlay =
+      (graphEventId != null && graphEventId.length > 0) || sessionViewId !== null;
```

Comment:
do we need `graphEventId.length > 0` ? Can we get away with `shouldShowOverlay = graphEventId || sessionViewId?`

Reply:
Can we unify the `!=` and `!==` approach in this statement just to use one? Could be `graphEventId` or `sessionViewId`  as `undefined`?

Reply:
I'm nervous to change it, for whatever reason, graph event id is set to empty string sometimes and is undefined at other times